terraform/modules/ecs/ecs.tf

```diff
@@ -92,6 +142,33 @@ data "template_file" "import_schools_task" {
 
 }
 
+/* vacancies_scrape task definition*/
+data "template_file" "vacancies_scrape_task" {
+  template = "${file(var.ecs_vacancies_scrape_task_definition_file_path)}"
+
+  vars {
```

Comment:
This is now the third place we are repeating our application variables. I wonder if there is anything we can do at this point to control this duplication and making adding or changing variables less painful.